cmd/extensions/main.go

```diff
@@ -241,6 +262,8 @@ func parseEnvFlags() config {
 	pflag.Int32(logSizeLimitMBFlag, 1000, "Log file size limit in MB")
 	pflag.String(logLevelFlag, viper.GetString(logLevelFlag), "Agones Log level")
 	pflag.Duration(allocationBatchWaitTime, viper.GetDuration(allocationBatchWaitTime), "Flag to configure the waiting period between allocations batches")
+	pflag.Int32(sleepPeriod, 15, "Time for SIGTERM handler to sleep for.")
```

Comment:
Any idea why we set defaults twice here? Shouldn't this be..

```
plfag.Int32(sleepPeriodFlag, viper.GetInt(sleepPeriodFlag), ...)
```

I see for other ints we're doing the same thing, but it doesn't make sense? (Maybe let's start a trend here and fix it for these.)

Regardless, the value here is mismatched from the value on line 246.

(But separately, as I mentioned above, this should be a duration.)